setup.cfg

```diff
@@ -1,11 +1,11 @@
 # configure options here to work for all sub-packages
 [flake8]
-ignore = E401 E402 E722 E741 F821 F841 E203, E266, E501, W503, F403, F401 E712 E713 F811
-max-line-length = 120
+ignore = E402 E722 E741 F821 F841 E203 W503 F401 E712 E713 F811
+max-line-length = 150
```

Comment:
Since black (in pre-commit) is forcing lines to be no more than 119 characters, I'm surprised you had to increase this.

Reply:
Yeah, these files fail without it: 
![Screenshot from 2022-09-20 15-14-34](https://user-images.githubusercontent.com/30890419/191344919-f07e2978-15c4-422e-8f80-a6ef7fa2d84f.png)

Reply:
Ah, I think they're all comments.